src/manageiq_client/api.py

```diff
@@ -230,7 +236,7 @@ def reload(self, expand=False):
             kwargs = {"expand": expand}
         else:
             kwargs = {}
-        self._data = self._api.get(self._href, **kwargs)
+        self._data = self._api.get(self._href, save_response=False, **kwargs)
```

Comment:
@cben You're right, it's hard to follow which response would be saved, I simplified it a bit.
My idea is that it's ok to save always the last response with the exception of "actions" (e.g. `api.collection.services.action.create()`). Here the user wants response from the action itself, not responses produced during result processing.